pkg_resources/__init__.py

```diff
@@ -1309,17 +1314,40 @@ def get_default_cache():
     named "Python-Eggs".
     """
     return (
-        os.environ.get('PYTHON_EGG_CACHE')
-        or appdirs.user_cache_dir(appname='Python-Eggs')
+            os.environ.get('PYTHON_EGG_CACHE')
+            or appdirs.user_cache_dir(appname='Python-Eggs')
     )
 
 
+def fc(x, replace_with='-', also_accept=None, also_refuse=None):
```

Comment:
This function really needs a docstring, especially because `fc` doesn't mean anything to me.